browser/ui/views/profiles/brave_profile_menu_view.cc

```diff
@@ -9,10 +9,16 @@
 #include <utility>
 
 #include "brave/browser/profiles/profile_util.h"
+#include "chrome/app/vector_icons/vector_icons.h"
 #include "chrome/browser/browser_process.h"
+#include "chrome/browser/profiles/profile.h"
 #include "chrome/browser/profiles/profile_attributes_entry.h"
 #include "chrome/browser/profiles/profile_manager.h"
 #include "chrome/browser/profiles/profile_window.h"
+#include "chrome/browser/ui/browser_finder.h"
+#include "chrome/grit/generated_resources.h"
+#include "components/vector_icons/vector_icons.h"
```

Comment:
are there existing deps for this and ui/base?

Reply:
@bridiver - here: https://github.com/brave/brave-core/blob/master/browser/ui/BUILD.gn#L220